app/processing/siti-agri/handle-known-defects/correct-invalid-due-dates/invalid-due-dates.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [{
+  invalid: '2022-07-01',
+  correction: '2022-10-05'
+}, {
+  invalid: '2022-08-01',
+  correction: '2022-11-05'
+}, {
+  invalid: '2022-09-01',
+  correction: '2022-12-05'
+}, {
+  invalid: '2022-10-01',
+  correction: '2023-01-05'
+}, {
+  invalid: '2022-11-01',
+  correction: '2023-02-05'
+}, {
+  invalid: '2022-12-01',
+  correction: '2023-03-05'
+}, {
+  invalid: '2023-01-01',
+  correction: '2023-04-05'
+}, {
+  invalid: '2023-02-01',
+  correction: '2023-05-05'
+}, {
+  invalid: '2023-03-01',
+  correction: '2023-06-05'
+}]
```

Comment:
Could you please confirm this is a one-off list? 

If there is possibility of the list growing or the dates changing, we may need an alternative implementation that will not require re-building the app whenever we add, remove or update date-data.

Reply:
It's a one off list.  This list contains every date that we know will be incorrect during the first annual schedule.

The problem should be fixed by the end of October, at which point we can revert this change.